security-baseline/rule-types/github/osps-do-01.yaml

```
@@ -0,0 +1,44 @@
+version: v1
+release_phase: alpha
+type: rule-type
+name: osps-do-01
+display_name: The project has mechanisms for public discussion
+short_failure_message: The project does not publicize mechanisms for public discussion.
+severity:
+  value: info
+context:
+  provider: github
+description: |
+  Projects should encourage open communication and collaboration
+  within the their community, enabling users to provide feedback and
+  discuss proposed changes or usage challenges.
+guidance: |
+  Enable "Issues" or "Discussions" via the
+  [Settings](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository)
+  page on GitHub.
+def:
+  in_entity: repository
+  rule_schema: {}
+  ingest:
+    type: git
```

This should be a repo ingest:

```suggestion
  ingest:
    type: rest
    rest:
      endpoint: "/repos/{{.Entity.Owner}}/{{.Entity.Name}}"
      parse: json
```